VAMobile/src/store/slices/lettersSlice.test.ts

```diff
@@ -207,9 +207,6 @@ context('letters', () => {
       await store.dispatch(getLetterBeneficiaryData())
       const actions = store.getActions()
 
-      const startAction = _.find(actions, { type: ActionTypes.LETTER_START_GET_BENEFICIARY_DATA })
```

Comment:
had to remove this part since the new timing of changing the loading variable to true after the api is called for the unit tests skipped this part. Wish we could mock a response time in order for this test to still work.

Reply:
removed